ui/apps/platform/src/sagas/networkSagas.js

```diff
@@ -77,14 +79,30 @@ function* getNetworkGraphs(clusterId, namespaces, query) {
         return;
     }
 
+    // TODO: consider changing the call signature in NetworkService module to take an object
+    // because the number of positional params for the functions whick get flows and policies has
+    // become unwieldy
+    // Example: when we added a specific deployment option for the new PF Network Graph
+    // it broke the old graph's network sagas.
+    const EMPTY_DEPLOYMENT_FILTER = [];
```

Comment:
nit: This variable looks different than the other ones so it looks a bit odd. Maybe we can do it in snake case like the others

Reply:
I should added `_CONSTANT` to the end, because it's just a constant to make these sagas keep working with the expanded service functions, until we retire the sagas.